.env.example

```diff
@@ -1,7 +1,27 @@
-export HOST=localhost # The default is usually localhost
-export DB_NAME=olympians # Create a database and use this
-export DB_USER=name # Use your postgres user. The default is usually postgres
-export DB_PASSWORD='' # Use your database password
-export SENDGRID_API_KEY='API_KEY here' #This is the sendgrid api key
-export EMAIL_FROM="test@test.com" # This is the email that is sending the email
-export URL="http://127.0.0.1:8000/" #The base url where the frontend app is hosted
\ No newline at end of file
+wexport DB_NAME=olympians
+export DB_USER=novak
+export DB_PASSWORD=novak254
+export HOST=''
+
+export SOCIAL_AUTH_GOOGLE_OAUTH2_KEY='65389949271-kspefjrqipki3svnec3dqcsmh92rdgt1.apps.googleusercontent.com'
```

Comment:
@kelvinndmo For the secret values, you can keep them hidden and replace these values with something like `your google auth key`

Reply:
updated @JoanNgatia .Thanks for noting.